server/src/user/user.service.ts

```diff
@@ -26,4 +26,91 @@ export class UserService {
       throw error;
     }
   }
+
+  async getTodos(userId: number, houseId: number) {
+    try {
+      // Find the house of the user
+      const house = await this.prisma.house.findFirst({
+        where: { id: houseId },
+        select: { name: true },
+      });
+
+      if (!house) {
+        throw new Error('House not found');
+      }
```

Comment:
You'll get 500 error in this case but you want to return 404 (not found status). Use `NotFoundException`!

Reply:
I fixed it!
https://github.com/HAK2024/fairy-share/pull/61/commits/5bfc635c690072ff657c9ef7682c2fc2a21fc9a8